eo-maven-plugin/src/main/java/org/eolang/maven/OptimizeMojo.java

```diff
@@ -259,7 +260,10 @@ private XML optimize(final Path file) throws FileNotFoundException {
                 new Rel(dir)
             );
         }
-        return new Xsline(trn).pass(new XMLDocument(file));
+        return new Xsline(
```

Comment:
@MikhailLipanin You can use the next code instead:
```java
        return new Xsline(
            new TrAfter(trn, new StClasspath("/org/eolang/parser/fail-on-critical.xsl"))
        ).pass(new XMLDocument(file));
```

Reply:
@volodya-lombrozo Yes, but the difference here in one thing:
Now it is done like this:
- All pipeline executed and stopped if it was necessary
- Then, only one time we apply `fail-on-critical`.
But using `TrAfter`, it will every time after each shift try to apply `fail-on-critical`, so the difference in number of attempts to apply `fail-on-critical`

Reply:
So, it is required to change this one?

Reply:
@MikhailLipanin I understood the difference. No, in that case it's better to leave the code as is.